packages/create-remix/templates/fly-stack/README.md

```diff
@@ -0,0 +1,85 @@
+# Remix Fly Stack
+
+- [Remix Docs](https://remix.run/docs)
+
+## Fly Setup
+
+1. [Install Fly](https://fly.io/docs/getting-started/installing-flyctl/)
+
+2. Sign up and log in to Fly
+
+```sh
+$ flyctl auth signup
+```
+
+3. You'll need to create a .env file in your project root for your SESSION_SECRET
+
+```sh
+$ cp .env.example .env
+$ echo SESSION_SECRET=$(openssl rand -hex 32) >> .env
+```
+
+## The Database
+
+In development, it's better to use a local database, The easiest way to do this is using [Docker][docker]. To start your postgres database, run the following command:
+
+```sh
+$ docker-compose up
+```
+
+That may take a moment to start up as it needs to get the postgres image from the Docker registry, after it's done, you'll need to migrate your database
+
+```sh
+$ prisma migrate deploy
+```
+
+If you'd prefer not to use Docker, you can also use Fly's Wireguard VPN to connect to a development database (or even your production database). You can find the instructions to set up Wireguard [here][fly_wireguard], and the instructions for creating a development database [here][fly_postgres].
+
+## Development
+
+From a new tab in your terminal:
+
+```sh
+$ npm run dev
+```
+
+This starts your app in development mode, rebuilding assets on file changes.
+
```

Comment:
Can you mention some of the features of the app and where to go to find different functionality. Probably just a couple paragraphs I think.